app/views/onboarding/NotificationDetails.tsx

```diff
@@ -3,22 +3,27 @@ import { useTranslation } from 'react-i18next';
 import { isGPS } from '../../COVIDSafePathsConfig';
 
 import { useStrategyContent } from '../../TracingStrategyContext';
-import OnboardingTemplate from './OnboardingTemplate';
+import DescriptionTemplate from '../common/DescriptionTemplate';
 
-const NotificationDetails = (props) => {
+interface NotificationDetailsProps {
+  navigation: any;
```

Comment:
I think its best to be explicit about the fact that we don't have any type safety around this.

Reply:
is that true though? could we be using the `useNavigation` hook?

Reply:
Ah I’ll check that out thanks!

Reply:
Do you have time to pair on this @johnschoeman?

Reply:
have a few more meetings today, but we can sync up early next week!